semgrep/semgrep/evaluation.py

```diff
@@ -137,39 +137,35 @@ def _evaluate_single_expression(
         raise UnknownOperatorError(f"unknown operator {expression.operator}")
 
 
-# Given a `where-python` expression as a string and currently matched metavars,
-# return whether the expression matches as a boolean
 def _where_python_statement_matches(
     where_expression: str, metavars: Dict[str, Any]
 ) -> bool:
     # TODO: filter out obvious dangerous things here
-    output_var = None
+    result = False
 
-    # HACK: we're executing arbitrary Python in the where-python,
```

Comment:
"Be careful" doesn't really tell me anything here - the `# nosem` below tells me this is "approved insecure behavior"